cogboard-webapp/src/components/DragCover/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import styled from '@emotion/styled/macro';
+
+const DragCard = ({ className }) => (
```

Comment:
Inconsistent name with the file name. Try to use already defined DStyledCard and modify properties there according to isDragging prop like in Widget component.